jwst/datamodels/schemas/slitmeta.schema.yaml

```diff
@@ -0,0 +1,100 @@
+%YAML 1.1
+---
+$schema: "http://stsci.edu/schemas/fits-schema/fits-schema"
+id: "http://stsci.edu/schemas/jwst_datamodel/slitmeta.schema"
+allOf:
+- type: object
+  properties:
+    name:
+      title: Name of the slit
+      type: string
+      fits_keyword: SLTNAME
+      fits_hdu: SCI
+    xstart:
+      title: Starting pixel in x direction
+      type: integer
+      default: 1
+      fits_keyword: SLTSTRT1
+      fits_hdu: SCI
+    xsize:
+      title: Number of pixels in x direction
+      type: integer
+      default: 0
+      fits_keyword: SLTSIZE1
+      fits_hdu: SCI
+    ystart:
+      title: Starting pixel in y direction
+      type: integer
+      default: 1
+      fits_keyword: SLTSTRT2
+      fits_hdu: SCI
+    ysize:
+      title: Number of pixels in y direction
+      type: integer
+      default: 0
+      fits_keyword: SLTSIZE2
+      fits_hdu: SCI
+    slitlet_id:
+      title: Slitlet ID
+      type: integer
+      default: 0
+      fits_keyword: SLITID
+      fits_hdu: SCI
+    source_id:
+      title: Source ID
+      type: integer
+      default: 0
+      fits_keyword: SOURCEID
+      fits_hdu: SCI
+    source_name:
+      title: Source name
+      type: string
+      fits_keyword: SRCNAME
+      fits_hdu: SCI
+    source_alias:
+      title: Source alias
+      type: string
+      fits_keyword: SRCALIAS
+      fits_hdu: SCI
+    stellarity:
+      title: Source stellarity
+      type: number
+      fits_keyword: STLARITY
+      fits_hdu: SCI
+    source_type:
+      title: Source type (point/extended)
+      type: string
+      fits_keyword: SRCTYPE
+      fits_hdu: SCI
+    source_xpos:
+      title: Source position in slit (x-axis)
+      type: number
+      default: 0.0
+      fits_keyword: SRCXPOS
+      fits_hdu: SCI
+    source_ypos:
+      title: Source position in slit (y-axis)
+      type: number
+      default: 0.0
+      fits_keyword: SRCYPOS
+      fits_hdu: SCI
+    shutter_state:
+      title: All (open and closed) shutters in a slit
+      type: string
+      default: ""
+      fits_keyword: SHUTSTA
+      fits_hdu: SCI
+    quadrant:
+      title: MSA quadrant for slit
+      type: integer
+    xcen:
+      title: Center of shutter in MSA coordinates (X dir)
+      type: integer
+    ycen:
+      title: Center of shutter in MSA coordinates (Y dir)
+      type: integer
+- $ref: variance.schema
+- $ref: pathlosscorr.schema
+- $ref: bunit.schema
+- $ref: photometry.schema
```

Comment:
`photometry.schema` and `wcsinfo.schema` are already included in the top-level schemas into which `slitmeta.schema` is being included, so we don't need/want them included here too. Same with `bunit.schema`, because the BUNIT keyword is already explicitly defined in the upper-level schemas.